### PR TITLE
Add file save to file sub keybinding

### PR DESCRIPTION
**Pull request by DanTheColoradan**
I find myself trying to use `SPC f s` all of the time. I also added `SPC f S` to save all files in the window.

### CODE REVIEW

File: src/cljs/proton/layers/core/keybindings.cljs [hunk 1]
```diff
@@ -71,6 +71,10 @@
                        :action "application:open-your-snippets"}}
                :f {:title "advanced-open-file"
                    :action "advanced-open-file:toggle"}
+               :s {:title "save file"
+                   :action "core:save"}
+               :s {:title "save all files"
```
Comment (spencerlyon2) [inline src/cljs/proton/layers/core/keybindings.cljs]: This looks like another lower case `s` instead of capital `S`

Comment (DanTheColoradan) [inline src/cljs/proton/layers/core/keybindings.cljs]: Thanks :)

**Comment by geksilla** [comment]
LGTM

**Comment by jessejanderson** [comment]
Glad I'm not alone in trying to use `SPC f s` all the time in Atom...

**Comment by DanTheColoradan** [comment]
Squashed into one commit.

**Pull request merged**